templates/ort-scan.yml

```diff
@@ -231,8 +235,12 @@
           - type: OrtConfig
             enabled: false
       EOF
+      else
+      echo -e "\e[1;33m Using 'config.yml' from 'ort-config' repository..."
       fi
-    
+      
+      ${ORT_CLI} config --show-active
```

Comment:
Should we always print the active ORT config - might contain sensitive data

Reply:
Looks like we have a mechanism for stipping it out. E.g. it does not show the password for the JDBC connection.